### PR TITLE
devel mode: Allow to skip assert/check_screen timeout

### PR DESCRIPTION
**Pull request by Martchus**
* See https://progress.opensuse.org/issues/44249
* Requires https://github.com/os-autoinst/os-autoinst/pull/1067

---

screenshot_20181127_141313

### CODE REVIEW

**Comment by coolo** [comment]
looks nice - and that it can be implemented in such short time proves we chose the right path.

**Comment by Martchus** [comment]
I now pushed 2 further commits to extend the fullstack tests. These really rely on changes in os-autoinst so the test might fail. (15 seconds for the assert_screen timeout is a bit less so there's a possibility to miss the time span in which it is possible to skip.)

**Comment by Martchus** [comment]
Can I merge this (together with the os-autoinst PR)?

@os-autoinst/tools-team Maybe someone wants to review this first?

File: t/33-developer_mode.t [hunk 1]
```diff
@@ -161,6 +161,7 @@ my $JOB_SETUP
   . 'FLAVOR=flavor BUILD=1 MACHINE=coolone QEMU_NO_TABLET=1 INTEGRATION_TESTS=1 '
   . 'QEMU_NO_FDC_SET=1 CDMODEL=ide-cd HDDMODEL=ide-drive VERSION=1 TEST=core PUBLISH_HDD_1=core-hdd.qcow2 '
   . 'TESTING_ASSERT_SCREEN_TIMEOUT=1';
+# setting TESTING_ASSERT_SCREEN_TIMEOUT is important here (see os-autoinst/t/data/tests/tests/boot.pm)
```
Comment (foursixnine) [inline t/33-developer_mode.t]: :+1:

**Comment by foursixnine** [comment]
Failing test is due to missing changes in os-autoinst

**Pull request merged**